python/ray/util/annotations.py

```diff
@@ -0,0 +1,82 @@
+def PublicAPI(*args, **kwargs):
+    """Annotation for documenting public APIs.
+
+    Public APIs are classes and methods exposed to end users of Ray. You
+    can expect these APIs to remain backwards compatible even across major Ray
+    releases.
+
+    Args:
+        stability: Either "stable" for stable features or "beta" for APIs that
```

Comment:
So for `stability="beta"` we are not guaranteeing that we won't change them, right?

Reply:
Of course